xlsx2csv.py

```diff
@@ -1202,12 +1196,11 @@ def main():
             if options.sheetname:
                 sheetid = xlsx2csv.getSheetIdByName(options.sheetname)
                 if not sheetid:
-                    raise XlsxException("Sheet '%s' not found" % options.sheetname)
+                    sys.exit("Sheet '%s' not found" % options.sheetname)
```

Comment:
If handled explicitly means something like `SheetNotFoundException` I could do that. I just thought that ince we're in `main` I'd use `sys.exit`

Reply:
I mean, it can use sys.exit there.